src/Symfony/Component/Process/Messenger/RunProcessMessage.php

```diff
@@ -22,11 +22,22 @@ public function __construct(
         public readonly ?array $env = null,
         public readonly mixed $input = null,
         public readonly ?float $timeout = 60.0,
+        public readonly ?string $commandLine = null,
```

Comment:
this shoudn't be exposed as a constructor argument

Reply:
Done, seems much cleaner this way.